src/biomappings/wsgi.py

```diff
@@ -55,6 +57,18 @@ def predictions(
             for line_prediction, _ in zip(it, range(limit)):
                 yield line_prediction
 
+    @staticmethod
+    def get_url(prefix: str, identifier: str) -> str:
+        """Return URL for a given prefix and identifier."""
+        if miriam_validator.namespace_embedded(prefix):
+            return f'https://identifiers.org/{identifier}'
+        else:
+            return f'https://identifiers.org/{prefix}:{identifier}'
+
+    def get_total(self) -> int:
```

Comment:
just a bit confusing since `total` is already an instance variable, I will change one of these to be a bit more specific

Reply:
Addressed in 0989970